apps/judicial-system/web/webpack.config.js

```diff
@@ -1,12 +1,9 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const TreatPlugin = require('treat/webpack-plugin')
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const nrwlConfig = require('@nrwl/react/plugins/webpack.js')
+const nrwlConfig = require('./../../../libs/shared/webpack/nrwl-config')
```

Comment:
Shouldn't we use @island.is/... when importing?

Reply:
Jubb definetely, VS Code auto import has been doing this trick on me, I will update this.

Reply:
Yes usually, but this is inside of `webpack.config.js` so it won't translate the `tsconfig.base.json` `paths` except if we would add usage of for example `tsconfig-paths-webpack-plugin`. But I will leave that for a separate discussion.